gabriel-portfolio/portfolio/src/main/webapp/style.css

```diff
@@ -113,15 +113,27 @@
   margin-left: 20px;
 }
 
+#name-box {
+  margin-left: 5px;
+}
 
-/* #enter-name {
-  display: inline-block;
-  margin-top: 20px;
+#comment-box-label {
+  font-size: 20px;
 }
 
-#nameBox {
-    height: 15px;
-} */
+.text-comments {
+  margin-bottom:5px;
```

Comment:
This is fine, but you could consider pulling out margin-bottom/margin-top to a common comment class since it's shared between text and date comments.

Reply:
I wanted to keep their styles separate because they are essentially 2 different objects, but they do share some styling aspects.